feature-libs/cart/added-to-cart-toast/assets/translations/en/added-to-cart-toast.i18n.ts

```diff
@@ -1,8 +1,10 @@
 export const addedToCartToast = {
-  entriesAdded: '{{ quantity }} items added',
-  entryAdded: '{{ quantity }} item added',
-  continueShopping: 'Continue Shopping',
-  goToCart: 'Go to Cart',
-  quantity: 'Qty',
-  price: 'Price',
+  addedToCartToast: {
```

Comment:
It looks like the key will have duplicate variable `addedToCartToast.addedToCartToast.price` please change the key